<launch>
  <!--若使用了bag的虚拟时间，use_sim_time可以防止rviz报错-->
  <param name="use_sim_time" value="false" />
  <!--调用平板控制指令的launch文件-->
  <node pkg="steer_track" type="tcp_server_node" name="tcp_server_node" respawn="false" output="screen" ns="robot_1">  
        <param name="SERVER_PORT"               type = "int"    value="9527"/>
  </node> 

  <!-- <node pkg="steer_track" type="tcp_server_node" name="tcp_server_node" respawn="false" output="screen" ns="gongzhuang">  
        <param name="SERVER_PORT"               type = "int"    value="9528"/>
  </node>    -->
  <!--若需要自动重启，设置respawn为true-->
  <node pkg="steer_track" type="wit_tcp_client_ros.py" name="imu" args="" output="screen" ns="robot_1">
      <param name="ip"               type = "str"    value="192.168.0.89"/>
      <param name="port"               type = "int"    value="23"/>
  </node>
  <!-- 控制任务和tcp通讯接口任务对应 python 文件 -->
  <node pkg="steer_track" type="task_node_old.py" name="task_node_old" output="screen" ns="robot_1">
    <rosparam file="$(find steer_track)/config/params.yaml" command="load"/>
  </node>

  <node pkg="steer_track" type="task_node.py" name="task_node" output="screen"  ns="robot_1"/>


    <!--调用发布定位信息的文件，若使用optitrack，其中关键变量为steerRobot，需要修改为刚体的名字-->
  <!-- <arg name="climbot_name" default="steer_tk"/> -->
  <!-- <node pkg="steer_track" type="pose_vel_pub_node" name="pose_vel_pub_node" args="$(arg climbot_name)" output="screen" ns="robot_1"/>   -->
  <!-- <node pkg="steer_track" type="motor_line" name="motor_line" output="screen"  ns="robot_1"/> -->

  <!-- <arg name="server" default="192.168.3.13"/> 
  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node> -->

</launch>



